config/database.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
   | interacting with SQL databases.
   |
   */
```

Comment:
@encodedBicoding do not set default DB settings from direct config they must inherit directly from `.env` file